repository/src/test/java/org/alfresco/schedule/AbstractScheduledLockedJobTest.java

```diff
@@ -70,7 +70,7 @@ public class AbstractScheduledLockedJobTest extends BaseSpringTest
     private SchedulerAccessorBean testCleanerAccessor;
     private JobDetail testCleanerJobDetail;
 
-    private static final Log LOGGER = LogFactory.getLog(AbstractScheduledLockedJobTest.class);
+    private static final Logger LOGGER = LoggerFactory.getLogger(AbstractScheduledLockedJobTest.class);
```

Comment:
If you're changing the logger then I suggest also changing the use of it:
```
LOGGER.info("Thread " + this.threadNum + " has started");
```
to
```
LOGGER.info("Thread {} has started", this.threadNum);
```

Reply:
Changed :) Thanks for pointing that one.